packages/eslint-plugin-calcite-components/src/rules/enforce-ref-last-prop.ts

```diff
@@ -0,0 +1,52 @@
+import { Rule } from "eslint";
+import type { JSXAttribute, JSXSpreadAttribute, JSXOpeningElement } from "@babel/types";
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      description: "Ensure that the ref attribute is placed as the last property in JSX elements to keep it up-to-date",
```

Comment:
Maybe:

` "Ensures the ref attribute is ordered last in a JSXElement to keep it up-to-date",`?